docs/document_loader.ipynb

```diff
@@ -321,15 +321,131 @@
    },
    "outputs": [],
    "source": [
-    "loader = AlloyDBLoader.create(\n",
+    "loader = await AlloyDBLoader.create(\n",
     "    engine,\n",
-    "    table_name=\"products\",\n",
+    "    table_name=TABLE_NAME,\n",
     "    content_columns=[\"product_name\", \"description\"],\n",
     "    format=\"YAML\",\n",
     ")\n",
     "docs = await loader.aload()\n",
     "print(docs)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create AlloyDBDocumentSaver"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBDocumentSaver\n",
+    "\n",
+    "# Creating a basic AlloyDBDocumentSaver object\n",
+    "saver = await AlloyDBDocumentSaver.create(\n",
+    "    engine,\n",
+    "    table_name=TABLE_NAME,\n",
+    "    content_column=\"product_name\",\n",
+    "    metadata_columns=[\"description\", \"content\"],\n",
+    "    metadata_json_column=\"metadata\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Save Documents to default table\n",
+    "The Alloy DB Document saver allows to save a list of Documents to the table using the first column as page_content and all other columns as metadata. \n",
+    "The default table will have the first column as page_content and the second column as metadata (JSON).\n",
+    "\n",
+    "Each row becomes a document.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_core.documents import Document\n",
+    "\n",
+    "test_docs = [\n",
+    "    Document(\n",
+    "        page_content=\"Red Apple\",\n",
+    "        metadata={\"description\": \"red\", \"content\": \"1\", \"category\": \"fruit\"},\n",
+    "    ),\n",
+    "    Document(\n",
+    "        page_content=\"Banana Cavendish\",\n",
+    "        metadata={\"description\": \"yellow\", \"content\": \"2\", \"category\": \"fruit\"},\n",
+    "    ),\n",
+    "    Document(\n",
+    "        page_content=\"Orange Navel\",\n",
+    "        metadata={\"description\": \"orange\", \"content\": \"3\", \"category\": \"fruit\"},\n",
+    "    ),\n",
+    "]\n",
+    "await saver.aadd_documents(test_docs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The added Document is returned with AlloyDBLoader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "loader = await AlloyDBLoader.create(engine, table_name=TABLE_NAME)\n",
```

Comment:
Can we add a note that we are using the table_name with the loader over the query to load the whole table?